### DISCUSSION TITLE
Custom dynamic slug

### DISCUSSION THREAD

**Discussion by netwaretcs** [statamic/cms#6248] [Q&A] [answered]
Hello, I would like to dynamically generate the slug using the unixtime instead of the title. Is there any way to do this?

**Comment by ryanmitchell** [comment]
You could listen to the EntrySaving event and update the slug in it?

**Comment by jasonvarga** [comment]
I was going to suggest putting `{{ date format="U" }}` in your route instead of the `{{ slug }}`, but it looks like we don't pass `date` in. We could do that.

**Comment by bencarr** [comment, marked as answer]
This is now possible in [Statamic v3.3.55](https://github.com/statamic/cms/releases/tag/v3.3.55) using [@jasonvarga's approach](https://github.com/statamic/cms/discussions/6248#discussioncomment-3031373). When setting up your route pattern, use the new `date` variable which provides the full `Carbon` instance for the entry date, and pass in whatever format you'd like to Antlers' `format` modifier:
```
/your/route/{{ date format="U" }}
```
